aws-cdk/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id("io.micronaut.build.internal.module")
+}
+
+dependencies {
+    api libs.aws.cdk.lib
+    api("io.micronaut.starter:micronaut-starter-api:$micronautVersion")
+    testImplementation(project(":function-aws-api-proxy"))
+}
+
+micronautBuild {
+    binaryCompatibility {
+        enabled = false
```

Comment:
Why disable it?

Reply:
Since `aws-cdk` is a new module, it fails unless you disable it. 

```
> Task :aws-cdk:japiCmp FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':aws-cdk:japiCmp'.
> Could not resolve all files for configuration ':aws-cdk:detachedConfiguration2'.
   > Could not find any matches for io.micronaut.aws:micronaut-aws-cdk:+ as no versions of io.micronaut.aws:micronaut-aws-cdk are available.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/io/micronaut/aws/micronaut-aws-cdk/maven-metadata.xml
     Required by:
         project :aws-cdk

* Try:
```